api/nakadi-event-bus-api.yaml

```diff
@@ -1270,3 +1281,43 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
```

Comment:
- failed/submitted is also fine for me, but you also have here ABORTED value. I think the main reason why we wanted a separate 'publish' field - is to be able to have a simple answer for a question "Was event published or not?"
-  ok, so for failed event we will also show the last step it passed, right?